backend/src/main/kotlin/org/loculus/backend/service/submission/ProcessedDataPostprocessor.kt

```diff
@@ -0,0 +1,19 @@
+package org.loculus.backend.service.submission
+
+import org.loculus.backend.api.Organism
+import org.loculus.backend.api.ProcessedData
+import org.springframework.stereotype.Service
+
+@Service
+class ProcessedDataPostprocessor(
```

Comment:
Would something like `ProcessedDataCompressionService` be a better name? I think that's more specific and explains better that it doesn't really change the data but only compresses them.

Reply:
I find`Compression` too misleading, because the metadata are not compressed. But in general I feel like this class could have a better name.

Reply:
They are not compressed into a binary (and not compressed very strongly) but at least in a broader sense, they are compressed, aren't they? But I can see that the name is also not perfect as people often tend to associate "compression" with other algorithms.

Reply:
The term "compression" is taken in software. I'd avoid that. That's why I like @anna-parker's suggestion of using "condensed".